tests/containers/bci_tests.pm

```diff
@@ -0,0 +1,149 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: bci-tests runner
+#   SUSE Linux Enterprise Base Container Images (SLE BCI)
+#   provides truly open, flexible and secure container images and application
+#   development tools for immediate use by developers and integrators without
+#   the lock-in imposed by alternative offerings.
+#   - install needed python environment
+#   - clone bci-tests repo
+#   - build project
+#   - run test for each given environment
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use XML::LibXML;
+use utils qw(systemctl zypper_call ensure_ca_certificates_suse_installed);
+use version_utils qw(get_os_release);
+use containers::common;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $runtime        = get_required_var('RUNTIME');
+    my $bci_tests_repo = get_required_var('BCI_TESTS_REPO');
```

Comment:
Also don't forget to add it to the variables list.

Reply:
This variable is used only in this test, there are plenty of variables that are not in mentioned variables.md, I'm not sure it's a good idea if it's used in single place. 

It's required cause the location might change and currently it's pointing to some suse internal repo. I would prefer not to hardcode links to gitlab.suse.de.

Reply:
I was following this comment, and I checked that neither PUBLIC_CLOUD_* variables are in variables.md

Reply:
Right, and those are used in several places :)

Reply:
Guys there is this file and it should contain all variables we use. That is the rule we simply have to follow.
If you introduce a variable and not include it there you create mess and work for others.
I don't consider that funny, I consider that as problem.

Regarding the `PUBLIC_CLOUD_` variables I plan to collect them as the last part of [poo#97742](https://progress.opensuse.org/issues/97742).

Reply:
I was thinking about this and:
What if we use this approach with `RUNTIME` everywhere?

Reply:
No one said it was funny :)

So, we agree that public cloud variables should end up in variables.md. I have created [poo#98292](https://progress.opensuse.org/issues/98292) for that so we don't forget.

But for this specific case, I don't think it makes sense. I have a section at the end of this file to explain all the variables because they are used only in this file. I'm not sure we have such strict rule, but I might be wrong.

Reply:
> I was thinking about this and:
> What if we use this approach with `RUNTIME` everywhere?

We should find an agreement about https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12842 fist. But maybe it makes sense, yes.